.changeset/thick-bees-approve.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit-labs/ssr': major
+---
+
+Allow SSR renderers to produce asynchronous values
```

Comment:
I need to add more information about the breaking change and how to upgrade (use collectResult or collectResultSync)